snforge_std/src/cheatcodes/tx_info.cairo

```diff
@@ -15,6 +16,13 @@ struct TxInfoMock {
     transaction_hash: Option<felt252>,
     chain_id: Option<felt252>,
     nonce: Option<felt252>,
+    // starknet::info::v2::TxInfo fields
+    resource_bounds: Option<Span<ResourceBounds>>,
```

Comment:
I wonder if someone would like to set only one field of the `ResourceBounds` struct. In such a definition, it is not possible. However, I'm not sure if that's the case. Just pointing

Reply:
Allowing for that would introduce changes in this `TxInfoMock` and the actual `TxInfo` struct, and the code would get more complicated. Plus, we don't even know if it's a use case. I'd stick to this version for now, although a valid comment 👍

Reply:
Fine, we can wait for some requests from the users 😌